apps/openmw_test_suite/CMakeLists.txt

```diff
@@ -1,59 +1,60 @@
+    mwworld/test_store.cpp
 
-        mwdialogue/test_keywordsearch.cpp
+    mwdialogue/test_keywordsearch.cpp
 
-        esm/test_fixed_string.cpp
+    esm/test_fixed_string.cpp
 
-        misc/test_stringops.cpp
+    misc/test_stringops.cpp
 
-        nifloader/testbulletnifloader.cpp
+    nifloader/testbulletnifloader.cpp
 
-        detournavigator/navigator.cpp
-        detournavigator/settingsutils.cpp
-        detournavigator/recastmeshbuilder.cpp
-        detournavigator/gettilespositions.cpp
-        detournavigator/recastmeshobject.cpp
-        detournavigator/navmeshtilescache.cpp
-        detournavigator/tilecachedrecastmeshmanager.cpp
+    detournavigator/navigator.cpp
+    detournavigator/settingsutils.cpp
+    detournavigator/recastmeshbuilder.cpp
+    detournavigator/gettilespositions.cpp
+    detournavigator/recastmeshobject.cpp
+    detournavigator/navmeshtilescache.cpp
+    detournavigator/tilecachedrecastmeshmanager.cpp
 
-        settings/parser.cpp
+    settings/parser.cpp
 
-        shader/parsedefines.cpp
-        shader/parsefors.cpp
-        shader/shadermanager.cpp
-    )
+    shader/parsedefines.cpp
+    shader/parsefors.cpp
+    shader/shadermanager.cpp
+)
 
-    source_group(apps\\openmw_test_suite FILES openmw_test_suite.cpp ${UNITTEST_SRC_FILES})
+source_group(apps\\openmw_test_suite FILES openmw_test_suite.cpp ${UNITTEST_SRC_FILES})
 
-    openmw_add_executable(openmw_test_suite openmw_test_suite.cpp ${UNITTEST_SRC_FILES})
+openmw_add_executable(openmw_test_suite openmw_test_suite.cpp ${UNITTEST_SRC_FILES})
 
-    target_link_libraries(openmw_test_suite ${GMOCK_LIBRARIES} components)
-    # Fix for not visible pthreads functions for linker with glibc 2.15
-    if (UNIX AND NOT APPLE)
-        target_link_libraries(openmw_test_suite ${CMAKE_THREAD_LIBS_INIT})
-    endif()
-
-    if (BUILD_WITH_CODE_COVERAGE)
-        add_definitions(--coverage)
-        target_link_libraries(openmw_test_suite gcov)
-    endif()
+target_link_libraries(openmw_test_suite ${GMOCK_LIBRARIES} components)
+# Fix for not visible pthreads functions for linker with glibc 2.15
+if (UNIX AND NOT APPLE)
+    target_link_libraries(openmw_test_suite ${CMAKE_THREAD_LIBS_INIT})
+endif()
 
-    if (MSVC)
-        if (CMAKE_CL_64)
-            # Debug version of openmw_unit_tests needs increased number of sections beyond 2^16
-            # just like openmw and openmw-cs
-            set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /bigobj")
-        endif (CMAKE_CL_64)
-    endif (MSVC)
+if (BUILD_WITH_CODE_COVERAGE)
+    add_definitions(--coverage)
+    target_link_libraries(openmw_test_suite gcov)
+endif()
 
+if (BULLET_USE_DOUBLES)
+    add_definitions(-DBT_USE_DOUBLE_PRECISION)
 endif()
+
+if (MSVC)
+    if (CMAKE_CL_64)
+        # Debug version of openmw_unit_tests needs increased number of sections beyond 2^16
+        # just like openmw and openmw-cs
+        set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /bigobj")
```

Comment:
This should probably be `add_target_compile_options_debug` or whatever so that it can't infect the other targets.

Reply:
This pr is not about fixing all possible problems. Only introduced change here is an indentation. Because `if (GTEST_FOUND AND GMOCK_FOUND)` is removed.

Reply:
This pr is not about fixing all possible problems. Only introduced change here is an indentation. Because `if (GTEST_FOUND AND GMOCK_FOUND)` is removed.